CapabilityStatements/CapabilityStatement-mp-MedicationData.RetrieveServe.Base.xml

```diff
@@ -27,6 +27,33 @@
   <rest>
     <mode value="client"/>
     <documentation value="Minimal requirements for a client to fulfill the 'Retrieve medication data' transaction (system role: MP-MGR)."/>
+    <resource>
```

Comment:
This CapabilityStatement is not implemented correctly, since each resource may only appear once in `.resource.type` (see the comment on https://hl7.org/fhir/r4/capabilitystatement-definitions.html#CapabilityStatement.rest.resource, which states 'Max of one repetition per resource type.'). Hence you have to combine all requirements for the different MedicationRequest resources/profiles, even though some of the SearchParameters and includes are only necessary for some of these profiles. The SearchParameter resources already state in which context they are necessary. Moreover the technical design (TO) can further specify for which profiles some of the SearchParameters and includes need to be supported. I'd use `.searchParam.documentation` to give additional guidance within this CapabilityStatement as well. For instance, for the reason search parameter you could add the following `.documentation`: 'This search parameter only needs to be supported for MedicationRequest resources that conform to the mp-MedicationAgreement and mp-VariableDosingRegimen profiles.'

Reply:
The `.searchInclude` has no dedicated `.documentation`, element, so for guidance on this I'd use the `.resource.documentation` element, or put the additional guidance in the corresponding `.searchParam.documentation` element, for instance 'This search parameter only needs to be supported for MedicationRequest resources that conform to the mp-MedicationAgreement and mp-VariableDosingRegimen profiles. In these cases the `_include` of the reason needs to be supported as well.'

Reply:
removed extra `.resource.type MedicationRequest` in server and client and added `searchParam` elements for reason and dispense-location. Also added `.searchParam.documentation`